service/src/main/java/uk/gov/hmcts/reform/fpl/model/tasklist/TaskSection.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.fpl.model.tasklist;
+
+import java.util.List;
+import java.util.Optional;
+
+public class TaskSection {
+
+    private final String name;
+    private final List<Task> tasks;
+    private String hint;
+    private String info;
+
+    private TaskSection(String name, List<Task> tasks) {
+        this.name = name;
+        this.tasks = tasks;
+    }
+
+    public static TaskSection newSection(String name, List<Task> tasks) {
+        return new TaskSection(name, tasks);
+    }
+
+    public TaskSection withHint(String hint) {
+        this.hint = hint;
```

Comment:
would @Getter and @Setter not remove need for some of these methods?

Reply:
The default setter does not enable chain setter methods, you have to do a bit more although if you wanted to you could add `@Accessor(chain = true)` I think

Reply:
yes, wanted to chain syntax, additionally wanted  getter that returns Optional for optional fields